### PR TITLE
Responsive Navbar

### PR DESCRIPTION
**Pull request by rojandahal**
-Responsive Navbar
-Edit responsive project section

### CODE REVIEW

File: src/assets/menu-hamburger.svg
Comment (aayussraut) [inline src/assets/menu-hamburger.svg:1]: make separate folder for svg

File: src/components/Menu/Menu.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+
+    // Remove event listener on cleanup
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+
+  function onClickHandler() {
+    toggle === "close" ? setToggle("open") : setToggle("close");
+    console.log(toggle);
```
Comment (aayussraut) [inline src/components/Menu/Menu.js:27]: yo hatau

File: src/components/Menu/Menu.js [hunk 2]
```diff
@@ -29,7 +52,40 @@ export default function Menu() {
...
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/about">About</Link>
+            </li>
+            <li>
+              <Link to="/">Services</Link>
+            </li>
+            <li>
+              <Link to="/">Blogs</Link>
```
Comment (aayussraut) [inline src/components/Menu/Menu.js:85]: array banayera map gara

File: src/components/Menu/Menu.js [hunk 3]
```diff
@@ -29,7 +52,40 @@ export default function Menu() {
             </ul>
           </div>
 
-          <div className="button">
+          <div className="nav-menu">
+            <img
+              src={menuIcon}
+              alt=""
+              className="hamburger"
+              onClick={onClickHandler}
```
Comment (aayussraut) [inline src/components/Menu/Menu.js:60]: onClick={()=>setToggle(prevValue=>!prevValue}

File: src/components/Menu/Menu.js [hunk 4]
```diff
@@ -1,13 +1,36 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./menu.css";
 import Button from "../Buttons/Button";
 import { Link } from "react-router-dom";
+import menuIcon from "../../assets/menu-hamburger.svg";
 
 export default function Menu() {
+  const [toggle, setToggle] = useState("close");
```
Comment (aayussraut) [inline src/components/Menu/Menu.js:8]: useState("false")

Comment (rojandahal) [reply src/components/Menu/Menu.js:8]: -I have implemented the state as a className. So if I change it to boolean value then the code won't be able to change the css for the navbar.
Should I change the css for it as well ??

File: src/components/Menu/Menu.js (hunk 3 above)
Comment (rojandahal) [reply src/components/Menu/Menu.js:60]: -Yes but it implements the state as className

**Review by aayussraut — APPROVED** [review]

**Pull request merged**